ui/kotlinx-coroutines-android/android-unit-tests/test/ordered/tests/CustomizedRobolectricTest.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package ordered.tests
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.test.*
+import org.junit.Test
+import org.junit.runner.*
+import org.robolectric.*
+import org.robolectric.annotation.*
+import org.robolectric.shadows.*
+import kotlin.test.*
+
+
+class InitMainDispatcherBeforeRobolectricTestRunner(testClass: Class<*>) : RobolectricTestRunner(testClass) {
+
+    init {
+        kotlin.runCatching {
+            GlobalScope.launch(Dispatchers.Main) {  } // touch Main, watch it burning
```

Comment:
Yes, because Robolectric reloads the class.
Nevertheless, I want to have a test which tries to init `HandlerDispatcher.kt` (at the moment when any attempt to call `Looper.getMainLooper()` triggers an exception) and only then initialize robolectric.